corehq/apps/case_search/utils.py

```diff
@@ -66,6 +67,11 @@ def _add_include_closed(self):
         if include_closed != 'True':
             self.search_es = self.search_es.is_closed(False)
 
+    def _add_query(self):
+        query = self.criteria.pop('_query', None)
```

Comment:
Are you using the `_` because you are worried `query` might conflict with a case property? Would you be able to call it `_xpath_query` (or just `xpath_query`) instead? The word `query` is a bit overloaded here, since technically everything you are doing here is with a "query".

Reply:
Makes sense, I'll change to `_xpath_query`